ui/main.qml

```diff
@@ -184,6 +196,14 @@ StatusWindow {
         setX(Qt.application.screens[0].width / 2 - width / 2);
         setY(Qt.application.screens[0].height / 2 - height / 2);
 
+        if (localAppSettings.appSizeInitialized) {
```

Comment:
@PascalPrecht `then` block is redundant here since those values will be already set in lines 35 and 36. So it's enough just this
```
if (!localAppSettings.appSizeInitialized) {
  width = Screen.desktopAvailableWidth - 125
  height = Screen.desktopAvailableHeight - 125
  localAppSettings.appSizeInitialized = true
}
```